ccp/modules/datashield-setup.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -e
+
+if [ "$ENABLE_DATASHIELD" == true ]; then
+  # HACK: This only works because exporter-setup.sh and teiler-setup.sh are sourced after datashield-setup.sh
+  if [ -z "${ENABLE_EXPORTER}" ] || [ "${ENABLE_EXPORTER}" != "true" ]; then
+    echo "The ENABLE_EXPORTER variable is either not set or not set to 'true'."
```

Comment:
Please improve the error message. I would suggest the following

`log WARN ENABLE_EXPORTER is not enabled, which is a dependency for DataSHIELD!`

Note: log WARN will run an echo. It might make sense to use error instead.

Reply:
Fixed in 896b24be9bfcf1ebdf9f5433b4c835b7fd63c529 and ec6f930